_articles/extra-drive.md

```diff
@@ -19,7 +19,7 @@ section: hardware-drivers
 
 ---
 
-If you've added a second drive to your computer when you originally ordered it, then it came pre-formatted and ready for use; no additional setup is required. The drive will be made available when you first open it up after each time you start your computer, and will remain mounted until the next start/boot or until you click the eject (⏏) icon. If you prefer to have the drive made available (mounted) automatically every time you start your computer, this is possible with some configuration changes.  
+If you've added a second drive to your computer when you originally ordered it, then it came pre-formatted and ready for use; no additional setup is required. The drive will be made available when you first open it up after each time you start your computer, and will remain mounted until the next start/boot or until you click the **eject (⏏) icon**. If you prefer to have the drive made available (mounted) automatically every time you start your computer, this is possible with some configuration changes.  
```

Comment:
This seems like a superfluous change, but the CONTRIBUTE.md file does state to bold the names of buttons, menu options, and other things that would be clicked.